womtool/src/main/scala/cromwell/pipeline/womtool/WomTool.scala

```diff
@@ -20,8 +20,9 @@ import wom.graph.{
 import wom.types.{ WomCompositeType, WomOptionalType, WomType }
 import womtool.input.WomGraphWithResolvedImports
 
-class WomTool(val importResolvers: List[ImportResolver]) extends WomToolAPI {
+class WomTool extends WomToolAPI {
 
+  private val importResolvers = DirectoryResolver.localFilesystemResolvers(None) :+ HttpResolver(relativeTo = None)
```

Comment:
Im not sure about this

Reply:
@myazinn @ilyaonishenko what do you think about moving this field into class? Do we need importResolvers somewhere else in project?

Reply:
Returned it back